data/hackathon/scenario7.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Accelerate GenAI experimentation by using an open source Enterprise-grade tool and software supply chain to get started
+exercise: 7
+date: '2024-06-25'
+tags: ['podman desktop','ai lab','genai', 'granite','ai applications']
+draft: false
+authors: ['default']
+summary: "Demonstrate Podman Desktops in-built"
+---
+
+ACME loved the flexibility at enterprise grade quality and scale of OpenShift AI and also the game changing approach to LLMs through InstructLab. 
+But all this excitement needs to be pointed in to the right direction with the right tooling right from when experimentation starts.
+"Can you help us?" they want to know - "I am glad you asked" is what you are saying.
```

Comment:
Adding some extra flavour in the scenario

```suggestion
The ACME Data Science team loved the flexibility, scale and enterprise grade quality of OpenShift AI. But all this excitement needs to be pointed in to the right direction with the right tooling right from when experimentation starts.

The ACME Senior Manager for Data Science Dave McDataScienceFace is concerned their data science team are still running ancient versions of docker desktop (`4.0.1` from 2021 🤮) and that every data scientist in the team seems to be spending weeks just getting basic local AI development environments running on their laptops.

The team workflows heavily rely on local docker containers for development so this seems like a great opportunity to push for ACME Financial Services to adopt 🌟 [Podman Desktop with Podman AI Lab](https://developers.redhat.com/articles/2024/05/07/introducing-podman-ai-lab) 🌟.

You've agreed to put together a quick demo to run for Dave and the Data Scientists at their upcoming team meeting. Can you secure agreement for the adoption of Podman Desktop? 

If you can pull this off Dave McDataScienceFace reckons this will accelerate how quickly the data science team are consuming ROSA compute time by getting models developed & deployed to OpenShift AI much more quickly.
```